src/Symfony/Component/Debug/Exception/FatalErrorException.php

```diff
@@ -9,28 +9,28 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\Debug\Exception;
-
-use Symfony\Component\HttpKernel\Exception\FatalErrorException as LegacyFatalErrorException;
+namespace Symfony\Component\HttpKernel\Exception;
```

Comment:
This diff is missleading - I didn't change the namespace, only moved these lines on to of the file when they were below previously